apps/wallet-mobile/src/features/Swap/common/SelectPool/SelectPoolFromList/SelectPoolFromList.tsx

```diff
@@ -69,9 +74,17 @@ export const SelectPoolFromList = ({pools = []}: Props) => {
                     <View>
                       <Spacer height={8} />
 
-                      <Boundary>
-                        <PriceInAda pool={pool} wallet={wallet} sell={orderData.amounts.sell} />
-                      </Boundary>
+                      <View style={styles.info}>
+                        <Text style={styles.infoLabel}>{strings.price}</Text>
+
+                        <Text style={styles.infoValue}>
+                          {Quantities.format(
+                            orderData.selectedPoolCalculation?.prices.market ?? Quantities.zero,
+                            denomination,
```

Comment:
does it handle negative denomination?

Reply:
Yes, negative denomination is fine, just moves the comma the other way.